packages/dev/inspector/tsconfig.build.json

```diff
@@ -17,7 +17,7 @@
             "shared-ui-components/*": ["dev/sharedUiComponents/dist/*"],
             "node-editor/*": ["tools/nodeEditor/dist/*"],
             "gui-editor/*": ["tools/guiEditor/dist/*"],
-            "viewer/*": ["tools/viewer/dist/*"],

```

Comment:
is there any reasons for this ?

Reply:
I guess I'm not sure if we need these anymore... @RaananW?